applications/OptimizationApplication/python_scripts/utilities/execution_policy_decorator.py

```diff
@@ -0,0 +1,88 @@
+import KratosMultiphysics as Kratos
+from KratosMultiphysics.model_parameters_factory import KratosModelParametersFactory
+from KratosMultiphysics.OptimizationApplication.execution_policies.execution_policy import ExecutionPolicy
+from KratosMultiphysics.OptimizationApplication.utilities.logger_utilities import FileLogger
+from KratosMultiphysics.OptimizationApplication.utilities.helper_utilities import OptimizationProcessFactory
+
+class ExecutionPolicyDecorator(ExecutionPolicy):
+    def __init__(self, model: Kratos.Model, parameters: Kratos.Parameters):
+        super().__init__()
+
+        default_parameters = Kratos.Parameters("""{
+            "execution_policy_name"    : "",
```

Comment:
look, here I strongly suggest removing repeated execution_policy_ and also reducing the number of input settings. The default settings should be to use Kratos solvers like structural or fluid  solvers which can be identified from the analysis input file. Otherwise, we will come up with a long input file which can be easily avoided.

Reply:
done :D